flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/AbstractBinaryWriter.java

```diff
@@ -141,6 +141,16 @@ public void writeRow(int pos, BaseRow input, BaseRowSerializer serializer) {
 		}
 	}
 
+	@Override
+	public void writeBinary(int pos, byte[] bytes) {
+		int len = bytes.length;
+		if (len <= 7) {
```

Comment:
how about replacing the magic number with a variable?

Reply:
OK